portfolio-website/app/components/About/About.tsx

```diff
@@ -0,0 +1,59 @@
+import getJsonFile from "@/app/utils/jsonFileReader"
+
+const About = async () => {
+  const [titles, buttons, about] = await Promise.all([
+    getJsonFile('titles'),
+    getJsonFile('buttons'),
+    getJsonFile('about')
+  ]);
+  
+  const birthDate = parseDate(about.data.ageValue);
+  const currentAge = calculateAge(birthDate);
+
+  function parseDate(dateStr: string): Date {
+    const parts = dateStr.split('.');
+    return new Date(parseInt(parts[2]), parseInt(parts[1]) - 1, parseInt(parts[0]));
+  }
+
+  function calculateAge(birthDate: Date): number {
+    const today = new Date();
+    let age = today.getFullYear() - birthDate.getFullYear();
+    const m = today.getMonth() - birthDate.getMonth();
+    if (m < 0 || (m === 0 && today.getDate() < birthDate.getDate())) {
+      age--;
+    }
+    return age;
+  }
```

Comment:
All of this should live in a utils function. 

Then the calculateAge will be exported and imported in this component, directly passing your own birth date to it ... you are not gonna change your birthdate before you die again anyway, so why would you need to pass an argument to it ? 

Then this function will be directly invoked into the currentAge span, no need to assign it to constant or state or whatever.